scripts/prepare.js

```diff
@@ -34,9 +34,16 @@ function cleanup() {
   if (fs.existsSync(path.join(process.cwd(), 'dist'))) {
     const files = shell.find('dist').filter((filePath) => {
       // Do not remove folder
-      // And do not clean anything for @storybook/cli/dist/generators/**/template* because these are the template files
+      // And do not clean anything for:
+      // - @storybook/cli/dist/generators/**/template*
+      // - @storybook/cli/dist/framework/*
+      // because these are the template files
       // that will be copied to init SB on users' projects
-      if (fs.lstatSync(filePath).isDirectory() || /generators\/.+\/template.*/.test(filePath)) {
+      if (
+        fs.lstatSync(filePath).isDirectory() ||
+        /generators\/.+\/template.*/.test(filePath) ||
+        /dist\/framework\/.*/.test(filePath)
```

Comment:
absolutely not sure about this line

Reply:
What is this line trying to do?

Reply:
Last build step is cleaning up folders.
Because cli/src/framework/* is meant to be copied and not compiled, it's in the original format in dist.
beacause it's .ts / .tsx, we need to tell `prepare` to not clean this folder.

The thing is prepare is valid for all projects so I can't only filter for `cli`